docs/modules/ROOT/pages/how-tos/http01-ocp.adoc

```diff
@@ -0,0 +1,22 @@
+= Use HTTP01 solver on OpenShift
+
+OpenShift mainly uses Route objects to configure the router.
+While it suppports Kubernetes Ingresses as well, they are translated into Routes in the background.
+However, cert-manager supports only Ingresses so we have to somehow "marry" them together.
+
+The main problem is that the generated Route for Let's Encrypt challenge resolver is redirecting HTTP traffic to HTTPS, thus the Let's Encrypt cannot reach the pod.
+In order to make provision Let's Encrypt certificates with the HTTP01 solver on OpenShift, you need to configure the solver like below:
+
+[source,yaml]
+----
+parameters:
+  cert_manager:
+    solvers:
+      haproxy_http01:
```

Comment:
I'm assuming that this name here doesn't actually play a role?

Reply:
No, that is irrelevant.